build.gradle

```diff
@@ -1869,7 +1874,8 @@ project(':clients') {
 
     testRuntimeOnly libs.jacksonDatabind
     testRuntimeOnly libs.jacksonJDK8Datatypes
-    testRuntimeOnly runtimeTestLibs
+    testRuntimeOnly libs.slf4jReload4j
```

Comment:
Could you please apply this approach on streams module too?

Reply:
I have applied to 4 modules which need JDK 11